Description: A patch for curve.c before 0.31.1-16
 This patch has been created to represent all changes was made direct
 to the package before implement dpkg-src 3.0(quilt) format, which has 
 a patch system.
 .
 Support for 64bits and remove some warnings.
Author: Edgar Antonio Palma de la Cruz <xbytemx@gmail.com>
Last-Update: 2011-08-07

--- autotrace-0.31.1.orig/curve.c
+++ autotrace-0.31.1/curve.c
@@ -116,7 +116,7 @@ log_curve (curve_type curve, at_bool pri
 
   if (!log_file) return;
 
-  LOG1 ("curve id = %x:\n", (unsigned long) curve);
+  LOG1 ("curve id = %lx:\n", (unsigned long) curve);
   LOG1 ("  length = %u.\n", CURVE_LENGTH (curve));
   if (CURVE_CYCLIC (curve))
     LOG ("  cyclic.\n");
@@ -177,7 +177,7 @@ log_entire_curve (curve_type curve)
 
   if (!log_file) return;
 
-  LOG1 ("curve id = %x:\n", (unsigned long) curve);
+  LOG1 ("curve id = %lx:\n", (unsigned long) curve);
   LOG1 ("  length = %u.\n", CURVE_LENGTH (curve));
   if (CURVE_CYCLIC (curve))
     LOG ("  cyclic.\n");
